ara/api/migrations/0012_playbook_usercontext.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-09-05 15:23
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0011_play_label_name_max_length'),
```

Comment:
Just a note so we don't forget about it -- this might change and needs rebasing depending on which one merges first, either this one or https://github.com/ansible-community/ara/pull/342 since it introduces a database migration too.

Reply:
Sure thing, just let us know once this PR is about to get merged.